wagtail/core/models.py

```diff
@@ -2738,6 +2742,11 @@ class Meta:
 class GroupApprovalTask(Task):
     groups = models.ManyToManyField(Group, verbose_name=_('groups'), help_text=_('Pages at this step in a workflow will be moderated or approved by these groups of users'))
 
+    admin_form_fields = Task.admin_form_fields + ['groups']
+    admin_form_widgets = {
```

Comment:
These (`admin_form_fields`, `admin_form_readonly_on_edit_fields`, `admin_form_widgets`) do need to be documented as they become integral to custom `Task` definition

`custom_tasks.rst` currently has references to `exclude_on_edit` which is now gone

Reply:
Forgot about that, thanks!

Docs are up to date now.